### ISSUE TITLE
feature request- Blendshapes

### ISSUE THREAD

**Issue by freekatet** [NickHardeman/ofxFBX#11]
Unless i missed, i don't see anything to control blend shapes?

**Comment by NickHardeman** [comment]
Correct, there is no dynamic control for blend shapes currently. But they should animate if provided in the fbx. They should update based on the animation? I don't have much experience with blend shapes...
https://github.com/NickHardeman/ofxFBX/blob/master/src/ofxFBXMesh.cpp#L708

**Comment by freekatet** [comment]
Thanks for the answer Nick.
The main idea would be to control the blend shape with OSC

**Comment by towooden** [comment]
Hi Nick,
Can you confirm that there is still no support for control of blend shapes? I can however see computeBlendShapes in ofxFBXMesh

**Comment by NickHardeman** [comment]
Hi @towooden,
Apologies for the super delayed response. I don't generally use blend shapes, so I can't even confirm what is in there works. I believe if you play the animation it would control the blend shape? It was adapted from the example code included in the FBX sdk. Looks like Arturo's add-on might let you control the blend shapes with parameters. 
https://github.com/arturoc/ofxFBX/blob/c717e3df957d208f41a7d5d33d490bfbbecca685/src/ofxFBXMesh.cpp#L163
